Cesium.Compiler/JsonObjectFile.cs

```diff
@@ -67,7 +67,7 @@ public override int GetHashCode()
         }
     }
 
-    public static bool IsCorrectExtension(LocalPath path) => path.GetExtensionWithDot() == ".obj";
+    public static bool IsCorrectExtension(LocalPath path) => path.GetExtensionWithDot() == ".obj" || path.GetExtensionWithDot() == ".o";
```

Comment:
I notice name of this function. Should be rename it to something more appropriate like `IsSupportedExtension`, or `IsObjectFileExtension` ?

Reply:
Yep, a great suggestion.